x-pack/filebeat/input/gcs/test/mock/data.go

```diff
@@ -50,6 +74,32 @@ var bucket_gcs_test_new = `{
 	"locationType": "region"
   }`
 
+var bucket_gcs_test_latest = `{
```

Comment:
s/var/const/ and for the same situations below. Alternatively (preferably IMO) they can just be string literals in the map where they are used to avoid the indirection and the non-idiomatic naming.

Reply:
sure